cmd/alertmanager/main.go

```diff
@@ -98,6 +102,8 @@ func newMarkerMetrics(marker types.Marker) {
 	prometheus.MustRegister(alertsSuppressed)
 }
 
+const defaultClusterAddr = "0.0.0.0:19093"
```

Comment:
9094 is the next free port in the core components range

Reply:
I felt like this was mostly for user-visible stuff. In mesh we just used the default as well.

Reply:
The default port list has expanded quite a bit since we first created it for a handful of official components. All our ports are there, including things like statsd ingestion so all things being equal we should stick to it.